### PR TITLE
Minor rebalance to traitor

### PR DESCRIPTION
**Pull request by Ar3nn**
siiiiiiiiiiiiiiiiiiiiiiiigh

:cl: Ar3nn
tweak: Wording and span changes to potential collaborator and codewords
/:cl:

### CODE REVIEW

**Comment by theColdflame** [comment]
:-1: to removing traitor infighting

:+1: to the rest of the changes, though

**Comment by Regen1** [comment]
:-1: 
I want traitors fighting

Nobody is safe

**Comment by Earthdivine** [comment]
Traitors are meant to be targeted by one another so you don't immediately trust an antag. Changlings used to be able to drain all of the DNA points out of another ling if they managed to catch them. It's a lot more nerve wrecking to see if the person being nice to you is out to kill you or not, but it is encouraged to do so with Hijacking and Collaborators.

Plus I'm sure it's also a small throw back to movie operations, kill the guy doing the job so no one knows who he's working for because you don't trust him.

**Comment by TheDZD** [comment]
```
code/game/gamemodes/traitor/traitor.dm:100:error: is_traitor: undefined var
code/game/gamemodes/traitor/traitor.dm:130:error: is_traitor: undefined var
code/game/gamemodes/traitor/traitor.dm:135:error: is_traitor: undefined var
code/game/gamemodes/traitor/traitor.dm:140:error: is_traitor: undefined var
code/game/gamemodes/traitor/traitor.dm:211:error: give_codewords: undefined proc
code/game/gamemodes/traitor/traitor.dm:326:error: give_codewords: undefined proc
```

**Comment by Lady-Luck** [comment]
Potential collaborator should be ditched altogether. :-1:

**Comment by Ar3nn** [comment]
Memes aside, your main issue with excluding traitors from each others objectives are that they shouldn't trust each other, but I think the current state of the game has evolved to the point that the station could handle it a good deal of the time, anyway. Admins prevent murderbone, so they can't just pool tc to get a bunch of chainsaws and stimpacks and go wild as an unstoppable wall of blades, unless they're trying to hijack, and even then, chainsaw sprees rarely go well once sec catches wind of it, so...I see no reason to continue this, personally. If its really such a big deal I'll put it back, but I'd like a better reason than memes and old prejudices that haven't had a chance to be looked at in years.

@Lady-Luck I've reworded potential collaborator to make it a bit more explicit that they aren't a dependable resource and should be avoided if possible.

**Comment by Lady-Luck** [comment]
I deleted my comment when I saw 'Removes Potential Collaborator text' but then I realized that was later changed.

I don't support this PR because it lends credibility to the potential collaborator system that should be ditched altogether. :-1:

**Comment by Ar3nn** [comment]
Lends credibility, nah. It takes away from it, because it uses more hmms and hahs that mean that they aren't sure. It gives a nudge towards the fact they could be wrong, and I specifically highlight that its for if they NEED another person's assistance.

**Comment by Lady-Luck** [comment]
Just remove it.

**Comment by Ar3nn** [comment]
The people that explained it to me did a pretty good job, and I'm not nearly as good at explaining stuff as they are, but the way it was explained to me makes me believe it can work better than you think it can, I just want to stress that it isnt a guarantee so people don't hunt this person down roundstart and demand help. If people don't do that, I think this system could work better.

**Comment by Lady-Luck** [comment]
The idea behind it is inherently flawed, and it wasn't even changed to be more reliable. Just because you opposed (or even worse, skeptical) toward NanoTrasen doesn't mean they're going to support someone akin to a terrorist.

It sucks, and trying to put a bandaid on it is just gonna make it stick around longer than it needs to.

**Comment by Ar3nn** [comment]
If I see more usage of it leading to dead tators, I'll remove it in my next PR, but I want to at least try to make it work, because it is a good system.

**Comment by Lady-Luck** [comment]
It's not going to be used any more or any less than it is now. New traitors are going to try it and get caught, just like always. Experienced traitors will simply ignore it, just like always.

**Comment by Ar3nn** [comment]
Experienced traitors don't ignore it, and I've had people tell me that they've worked with people like this on stuff like steal objectives, by simply making them bribe them with dosh, which encourages roleplay.

**Comment by Lady-Luck** [comment]
Kay.

**Comment by Ar3nn** [comment]
Also, I removed the dumb codewords, because people try to use those on the potential collaborator, which is what gets them caught because clunky, stupid conversation pieces.

**Comment by Earthdivine** [comment]
I'm sorry, I didn't know conversations about John Taylor sipping Martini's at a bank was grounds for being arrested.

Yea code words need to go no matter what.

**Comment by Ar3nn** [comment]
If the person and sec were metagamey enough they'd keep the guy under watch for it (an officer on his tail like always and ever) and then he'd be fuckeroo'd.

**Comment by Fox-McCloud** [comment]
Hell no. This is the only guaranteed way that traitors have of identifying themselves without showing their uplink or whipping out traitor-only gear.

**Comment by Lady-Luck** [comment]
It'd be better if the codewords should stay and the potential collaborator stuff should go, not the other way around.

**Comment by Ar3nn** [comment]
So, we don't want to remove traitors from traitor objective targets, but we want them to be able to identify each other. You people make no sense.

**Comment by Fox-McCloud** [comment]
>It'd be better if the codewords stayed and the potential collaborator stuff went, not the other way around.

This, if one must be removed. I've definitely seen RP opportunities come up a number of times from the potential collaborator thing though, so I dunno about removing it....I've seen a lot of people get turned in, too.

**Comment by tigercat2000** [comment]
how about we don't remove either and just clarify that they are unreliable 
so people can keep using this if they want and also understand it's a crapshoot

**Comment by Fox-McCloud** [comment]
Yeah, kinda what I'm inclined to say.

**Comment by Ar3nn** [comment]
@Fox-McCloud I know I already tossed it out, but what were your feels on the tator exclusion? Like, "this is never ever ever happening" like apparently everyone else says?

**Comment by Lady-Luck** [comment]
I just don't like what a huge newbie trap the potential collaborator system is. I feel like even with proper phrasing, it's gonna get overlooked and people will still fall into the trap.

**Comment by Ar3nn** [comment]
Yes, but maybe a few less people. It's entirely their fault if they overlook all the maybe's in it now, I literally could not make it anymore ambiguous without putting it in a warning class for no reason.

**Comment by Ar3nn** [comment]
Also, fine, whatever, fuck I'll put back the codewords, I didn't know people actually liked those, I've never heard a good thing said about them. The opinions that come crawling out of the woodwork on some of these PRs.

**Comment by Lady-Luck** [comment]
How about improving the codewords themselves? I've gotten some real oddballs before.

**Comment by Ar3nn** [comment]
I could probably improve on the selection, make up some things that sound more natural to talk about on a space station than John Travolta and a martini. But I'll save that for when I'm feeling more creative (meaning another PR.)

**Comment by Fox-McCloud** [comment]
You...still didn't re-include them.

**Comment by Ar3nn** [comment]
I had coded their generation back but forgot the distribution procs. Whoops.

File: code/game/gamemodes/traitor/traitor.dm [hunk 1]
```diff
@@ -332,13 +332,13 @@
 
 			to_chat(traitor_mob, "The Syndicate have cunningly disguised a Syndicate Uplink as your [R.name] [T.loc]. Simply enter the code \"[pda_pass]\" into the ringtone select to unlock its hidden features.")
 			traitor_mob.mind.store_memory("<B>Uplink Passcode:</B> [pda_pass] ([R.name] [T.loc]).")
-	if(!safety)//If they are not a rev. Can be added on to.
-		give_codewords(traitor_mob)
+		if(!safety)
```
Comment (Fox-McCloud) [inline code/game/gamemodes/traitor/traitor.dm]: you tabbed this farther than it was originally.

Comment (Ar3nn) [inline code/game/gamemodes/traitor/traitor.dm]: whoops

**Comment by Earthdivine** [comment]
I would like to add to the whole "Traitors shouldnt target one another", that we have limited amount of certain objective items, the antique gun, the Krag gloves, the HoS gun. Even if you remove the kill objectives directly, traitors would still kill traitors due to conflicting objectives.

**Comment by Ar3nn** [comment]
I already dropped the issue, but I didn't take the big step of just making their objectives a pool and letting them do it all as a team because I figured that was too far to say "Teamwork!" in this case. At this point I see no reason to revisit the issue because nobody else thinks they're weak enough yet, when as I have stated elsewhere, Space Law and SOP keep getting buffed to counter them because "loopholes r bad, mkay" even when the loophole is so tiny that them needing to loophole it is like every 30th round or longer between, and yet it was such a decisive issue that admins allowed security to loophole them into jail, because "antags cant win the interrogation, this is just not right" or something. It just feels incredibly rigged against antags at this point. It genuinely feels like only the extremely robust vets that have played for years and antag consistently can even have a shot at winning, yet they aren't weak enough to be buffed in any way. So, these are my feelings on the subject, but from the looks of this PR nobody agrees with it, so as I said, I'm dropping the issue.

**Comment by Fox-McCloud** [comment]
After talking this over with @TheDZD and more carefully reading what the text says---it already seems cautionary enough--this is...well, a bit crazy in how cautionary it is; especially an RP based feature.

**Pull request closed without merging**